receiver/mysqlreceiver/README.md

```diff
@@ -1,8 +1,12 @@
 # MySQL Receiver
 
-This receiver queries MySQL's global status and InnoDB tables.
+| Status                   |           |
+| ------------------------ |-----------|
+| Stability                | [beta]    |
```

Comment:
Please remove line 11, which also states the status.

Reply:
done